### PR TITLE
Correct the documentation describing the responses.

### PR DESCRIPTION
**Pull request by servingUpAces**
The documentation needed an update for the responses.

### CODE REVIEW

File: DOCUMENTATION.md [hunk 1]
```diff
@@ -494,12 +504,28 @@ You can choose first or second class postage for your precompiled letter. Set th
 
 ### Response
 
-If the request to the client is successful, the client returns a `LetterNotificationResponse` with the `id`, `reference` and `postage` set:
+If the request to the client is successful, the client returns a `LetterNotificationResponse` with the `id`, `reference` and `postage` sets the other attributes are null:
```
Comment (jonathanglassman) [inline DOCUMENTATION.md]: I don't understand this sentence. Does this mean that the `id`, `reference` and `postage` attributes are set to the relevant value, and the other attributes are set to null? Or that all attributes are set to null?

Comment (servingUpAces) [reply DOCUMENTATION.md]: Ah quite right - that doesn't sound right.

Comment (servingUpAces) [reply DOCUMENTATION.md]: It that better?

Comment (jonathanglassman) [reply DOCUMENTATION.md]: 👍 LGTM

**Pull request merged**